src/test/groovy/org/mockito/release/notes/vcs/IgnoredCommitTest.groovy

```diff
@@ -0,0 +1,24 @@
+package org.mockito.release.notes.vcs
+
+import spock.lang.Specification
+
+class IgnoredCommitTest extends Specification {
+
+    def "should skip commits with that contains given ignored substrings"() {
+        def ignoredCommit = new IgnoredCommit(["[ci skip]", "[custom skip]"])
+        def commitWithCiSkip = new GitCommit("firstCommitId", "sample@email.com", "sampleAuthor", "sample [ci skip] commit message")
+        def commitWithCustomSkip = new GitCommit("thirdCommitId", "sample@email.com", "sampleAuthor", "sample [custom skip] commit message")
+
+        expect:
+        ignoredCommit.isTrue(commitWithCiSkip)
+        ignoredCommit.isTrue(commitWithCustomSkip)
+    }
+
+    def "should not skip commit that does not contain given ignored substrings"() {
+        def ignoredCommit = new IgnoredCommit(["[ci skip]", "[custom skip]"])
```

Comment:
Second method uses the same IgnoredCommit instance so I think only one test method would be better. Especially since this is a quite straightforward logic and only parameter that changes (and matters) is message content.